features-json/input-datetime.json

```
@@ -417,7 +421,8 @@
     "2":"Partial support in iOS Safari refers to not supporting the `week` input type, nor the `min`, `max` or `step` attributes",
     "3":"Some modified versions of the Android 4.x browser do have support for date/time fields.",
     "4":"Can be enabled in Firefox using the `dom.forms.datetime` flag.",
-    "5":"Partial support refers to supporting `date` and `time` input types, but not `datetime-local`, `month` or `week`."
+    "5":"Partial support refers to supporting `date` and `time` input types, but not `datetime-local`, `month` or `week`.",
+    "6":"Partial support in macOS Safari refers to not supporting the `week` and `month` input type",
```

That trailing comma might me an issue in JSON? 0:-)

Not sure why, but the automatic checks don't seem to run anymore... @Fyrd?

```suggestion
    "6":"Partial support in macOS Safari refers to not supporting the `week` and `month` input type"
```